media/specs/manager.yaml

```
@@ -688,17 +700,34 @@ components:
         - directory
         - service
         - delegator
-    outway:
-      description: The details of the Outway for which a connection authorization was granted
+    outwayPublicKeyThumbprint:
+      description: The details of the Outway based on a public key thumbprint for which a connection authorization was granted
       type: object
       properties:
+        type:
+          $ref: "#/commponents/schemas/outwayType"
         peer_id:
           $ref: "#/components/schemas/peerID"
         public_key_thumbprint:
           $ref: "#/components/schemas/publicKeyThumbprint"
       required:
+        - type
         - peer_id
         - public_key_thumbprint
+    outwayDomainName:
+      description: The details of the Outway based on a domain name for which a connection authorization was granted
```

```suggestion
      description: The details of the Outway based on a domain name for which a connection was authorized
```

I think this flows better but only if 'connection authorization' is not a defined term already.